### ISSUE TITLE
Automated a11y tests

### ISSUE THREAD

**Issue by silentHoo** [inovex/elements#325]
Checking all interactive components for usability using keyboard navigation. Integration of an automated A11y test tool such as https://www.deque.com/axe/devtools/.

**Conspicuous components**
* ino-menu (close with esc)

**Acceptance criteria**
* Comparison of the currently available A11y test tools and their feature set. Orientation should be based on the GitHub stars.
* Decision for a suitable tool
* Creating the pipeline configuration
* Pipeline should run whenever our tests are running
* Creating an initial test case that will run successfully

**Comment by silentHoo** [comment]
I created some smaller chunks for that issues to make it easier for the developers to handle. Now we've the milestone https://github.com/inovex/elements/milestone/4.
